PW.07/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace PW._07
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+
+        public string[] EmptyArray(string[] arr)
+        {
+            string[] emptyArr = new string[0];
+            return emptyArr;
+        }
+
+        public string[] TriplArray(string[] arr)
+        {
+            int num = 32;
+            char leter = 'a';
+            string word = "Hello!";
+            SomeType[] triplArr = new string[] {num, leter, word };
```

Comment:
SomeType - what is it?

Reply:
here should be the Object type:

`object[] triplArr = new object[] {num, leter, word };`